### PR TITLE
[python] Trivial comments for wheel-build

### PR DESCRIPTION
**Pull request by johnkerl**
For #675 

This PR has become trivial now that it's rebased on the now-merged #677

### CODE REVIEW

**Review by thetorpedodog — APPROVED** [review]

**Comment by johnkerl** [comment]
This PR is comments-only and we have multiple CI jobs queued up waiting for resources. Letting full CI run on this PR is not a good use of resources, and is holding back other, more important things. Merging.

File: .github/workflows/py-packaging.yaml [hunk 1]
```diff
@@ -26,6 +29,7 @@ jobs:
         path: apis/python/dist/
 
   wheels:
+    # Note: tries all supported Python versions as specified in apis/python/pyproject.toml
```
Comment (gsakkis) [inline .github/workflows/py-packaging.yaml:32]: > as specified in apis/python/pyproject.toml

\<pedantic\>Currently this is actually specified in `setup.py` as `python_requires`\</pedantic\>

Comment (johnkerl) [reply .github/workflows/py-packaging.yaml:32]: @gsakkis thanks for the update! :)

Comment (johnkerl) [reply .github/workflows/py-packaging.yaml:32]: https://github.com/single-cell-data/TileDB-SOMA/commit/e5045e88412ff0c439c0c00bd6ffd795917846cf

**Pull request merged**